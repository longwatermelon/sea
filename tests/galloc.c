int arr = galloc(2,int);

int main() {
    *arr = 8;
    *(arr+8)=10;
    *(arr+8) = *arr + *(arr+8)*2;
    return *(arr+8);
}
