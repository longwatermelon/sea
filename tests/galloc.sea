let arr:int* = galloc(2,int);

fn main() -> int {
    arr[0]=8;
    arr[1]=10;
    arr[1] = arr[0] + arr[1]*2;
    return arr[1];
}
