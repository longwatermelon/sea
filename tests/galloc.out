RETURN_CODE:28
STDOUT:
